.github/workflows/pypi_publish.yml

```diff
@@ -12,26 +13,36 @@ on:
 
 jobs:
   build-sdist:
-    name: Build package
+    name: Build sdist
     runs-on: ubuntu-latest
     steps:
       - name: 💻 Checkout the repo
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true # include tags to get correct version from setuptools_scm
 
       - name: 🐍 Set up Python
-        uses: actions/setup-python@v4
+        uses: conda-incubator/setup-miniconda@v3
         with:
-          python-version: '3.11'
+          miniconda-version: "latest"
+          activate-environment: build-vhdf
+          environment-file: environment.yml
+          conda-remove-defaults: true
```

Comment:
What is the purpose of building the source tarballs with conda (which should have no dependencies other than `build` and `twine`) and everything else with pip? It looks like it's just adding extra complexity to me.

Reply:
Sure, I'll make this pure python and get rid of anything related to conda.